From 5bfa1afd1b9152e26410be59d737c58a713da187 Mon Sep 17 00:00:00 2001
From: Fabian Vogt <fvogt@suse.de>
Date: Tue, 22 Jun 2021 21:35:03 +0200
Subject: [PATCH] Revert "Bump version"

This reverts commit 8bc2ee81eb399a22cd2641aaab169c632dc94fe2.
---
 .qmake.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.qmake.conf b/.qmake.conf
index 3dea105..23f20e2 100644
--- a/.qmake.conf
+++ b/.qmake.conf
@@ -2,4 +2,4 @@ load(qt_build_config)
 CONFIG += warning_clean
 DEFINES += QT_NO_FOREACH
 
-MODULE_VERSION = 5.15.3
+MODULE_VERSION = 5.15.2
-- 
2.20.1

